h/static/scripts/login-forms/components/test/LoginForm-test.js

```diff
@@ -0,0 +1,128 @@
+import { checkAccessibility, mount } from '@hypothesis/frontend-testing';
+import { act } from 'preact/test-utils';
+
+import { Config } from '../../config';
+import { routes } from '../../routes';
+import { $imports, default as LoginForm } from '../LoginForm';
+
+describe('LoginForm', () => {
+  let fakeConfig;
+
+  beforeEach(() => {
+    fakeConfig = {
+      csrfToken: 'fake-csrf-token',
+      formData: {
+        username: '',
+        password: '',
+      },
+      formErrors: {},
+    };
+  });
+
+  afterEach(() => {
+    $imports.$restore();
+  });
+
+  const getElements = wrapper => {
+    return {
+      form: wrapper.find('form[data-testid="form"]'),
+      csrfInput: wrapper.find('input[name="csrf_token"]'),
+      usernameField: wrapper.find('TextField[name="username"]'),
+      passwordField: wrapper.find('TextField[name="password"]'),
+      forgotPasswordLink: wrapper.find('a[data-testid="forgot-password-link"]'),
+      submitButton: wrapper.find('Button[data-testid="button"]'),
+    };
+  };
+
+  const createWrapper = () => {
+    const wrapper = mount(
+      <Config.Provider value={fakeConfig}>
+        <LoginForm />
+      </Config.Provider>,
+    );
+    const elements = getElements(wrapper);
+    return { wrapper, elements };
+  };
+
+  it('renders the login form', () => {
+    const { elements } = createWrapper();
+    const {
+      form,
+      csrfInput,
+      usernameField,
+      passwordField,
+      forgotPasswordLink,
+    } = elements;
+
+    assert.equal(form.prop('action'), routes.login);
+    assert.equal(form.prop('method'), 'POST');
```

Comment:
We usually skip testing of static JSX (`method="POST"` here) and only test the dynamic parts.